src/crafting_gui.cpp

```diff
@@ -533,12 +534,38 @@ const recipe *select_crafting_recipe( int &batch_size )
                                         _( "Batch time savings: <color_cyan>%s</color>" ),
                                         current[line]->batch_savings_string() );
 
+                const int makes = current[line]->makes_amount();
+                if ( makes > 1 ) {
+                    ypos += fold_and_print( w_data, point( xpos, ypos ), pane, col,
+                        _( "Recipe makes: <color_cyan>%d</color>" ),
+                        makes );
+                }
+
                 print_colored_text(
                     w_data, point( xpos, ypos++ ), col, col,
                     string_format( _( "Dark craftable?  <color_cyan>%s</color>" ),
                                    current[line]->has_flag( flag_BLIND_EASY ) ? _( "Easy" ) :
                                    current[line]->has_flag( flag_BLIND_HARD ) ? _( "Hard" ) :
                                    _( "Impossible" ) ) );
+                
+                std::string nearby_string;
+                const int nearby_amount = crafting_inv.count_item( current[line]->result() );
+
+                if ( nearby_amount == 0 ) {
+                    nearby_string = "<color_light_gray>0</color>";
+                }
+                else {
+                    if ( nearby_amount > 9000 ) {
+                        // at some point you get too many to count at a glance and just know you have a lot
+                        nearby_string = _( "<color_red>It's Over 9000!!!</color>" );
+                    }
+                    else {
+                        nearby_string = string_format( "<color_yellow>%d</color>", nearby_amount );
+                    }
+                }
+                ypos += fold_and_print( w_data, point( xpos, ypos ), pane, col,
+                    _( "Nearby: <color_cyan>%s</color>" ), nearby_string );
```

Comment:
I think just "Nearby" is a little ambiguous, and there's certainly space in the UI to clarify a little more.  I'm not sure how best to tweak it; maybe "Already present nearby: %s" or "There are already %s nearby"?

Also, it seems like the `color_cyan` tag is redundant here; there's always another colour inside it.

Reply:
cyan tag is a result of me not paying attention to copy and paste when I rewrote it.

I struggled with what the label should be but ended up deciding to be as general as possible because I didn't want to get caught up with amount/charges/portions discrepancies.

Reply:
I think you need to at least make it clear that it's the recipe output that's being counted here, otherwise it's just an arbitrary number without context.